source/index.html

```diff
@@ -1,12 +1,157 @@
-<html lang="ru">
+<html  class="page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Кэт энерджи</title>
   </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
+  <body class="page__body">
+    <header class="main-header">
+      <a class="main-header__logo">
+        <img src="img/logo-desktop-1.svg" width="202" height="58" alt="Кэт Энерджи">
+      </a>
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list site-list">
+            <li class="site-list__item site-list__item--active">
+              <a href="#">
+                Главная
+              </a>
+            </li>
+            <li class="site-list__item"><a href="#">Каталог продукции</a></li>
+            <li class="site-list__item"><a href="#">Подбор программы</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main class="main-container">
+      <h1 class="visually-hidden">Функциональное
+        питание для котов</h1>
+        <section class="program container">
+          <h2 class="program__title">Функциональное
+            питание для котов</h2>
+            <p class="program__text">Занялся собой? Займись котом!</p>
+            <img class="program__image" src="img/index-can-mobile-1.png" width="280" height="270" alt="Cat Energy Pro курицa">
+            <a class="program__link" href="form.html">Подобрать программу</a>
+        </section>
+        <section class="categories container">
+          <h2 class="categories__header visually-hidden">индивидуальный план для кота</h2>
+          <ul class="categories__list">
+            <li class =" categories__item categories__item--slim">
+              <h3 class="categories__title">Похудение</h3>
+              <p class="categories__description">Ваш кот весит больше собаки и почти утратил способность лазить по деревьям? Пора на диету! Cat Energy Slim поможет вашему питомцу сбросить лишний вес.</p>
+              <a class="categories__link" href="#" aria-label="Каталог Slim" title="Каталог Slim">Каталог Slim</a>
+            </li>
+            <li class="categories__item categories__item--muscles">
+              <h3 class="categories__title">Набор массы</h3>
+              <p class="categories__description">Заработать авторитет среди дворовых котов и даже собак? Серия Cat Energy Pro поможет вашему коту нарастить необходимые мышцы!</p>
+              <a class="categories__link" href="#" aria-label="Каталог Pro" title="Каталог Pro">Каталог Pro</a>
+            </li>
+          </ul>
+        </section>
+        <section class="advantages container">
+          <h2 class="advantages__title"> Как это работает</h2>
+          <ul class="advantages__list">
```

Comment:
На других разрешениях есть цифры, так что предположим что порядок важен
```suggestion
          <ol class="advantages__list">
```